service.go

```diff
@@ -453,6 +454,7 @@ func DefaultErrorHandler(ctx context.Context, rw http.ResponseWriter, req *http.
 	} else {
 		Log.Error(ctx, e.Error())
 	}
+	go IncrCounter([]string{"goa", "handler", "default", "error", strconv.Itoa(status)}, 1.0)
```

Comment:
I would move these to were the error handler is invoked instead so that the counter gets incremented even if a user has specified a custom error handler, that is here: https://github.com/goadesign/goa/blob/master/service.go#L387